### PR TITLE
Added restrictAccess flag to enable/disable behavior of botkube. 

### PR DESCRIPTION
**Pull request by mahendrabagul**
This flag will control user's access to botkube via configured channel. If set to true, it will allow user to talk to botkube only through configured channel. In other case, botkube will be accessible from all channels.

Signed-off-by: Mahendra Bagul <mahendra@infracloud.com>

##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

Fix for https://github.com/infracloudio/botkube/issues/235

### CODE REVIEW

File: pkg/execute/executor.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ const (
 	notifierStartMsg   = "Brace yourselves, notifications are coming from cluster '%s'."
 	notifierStopMsg    = "Sure! I won't send you notifications from cluster '%s' anymore."
 	unsupportedCmdMsg  = "Command not supported. Please run /botkubehelp to see supported commands."
+	restrictAccessMsg  = "Sorry, You can use botkube from configured channel only."
```
Comment (PrasadG193) [inline pkg/execute/executor.go]:
```suggestion
	restrictAccessMsg  = "Sorry, You can use BotKube from configured channel only."
```

File: pkg/execute/executor.go [hunk 2]
```diff
@@ -217,9 +225,6 @@ func runKubectlCommand(args []string, clusterName string, isAuthChannel bool) st
 		}
 		finalArgs = append(finalArgs, arg)
 	}
-	if isAuthChannel == false {
-		return ""
-	}
```
Comment (PrasadG193) [inline pkg/execute/executor.go]: Please verify if `BotKube notifier` and `BotKube filters` commands are working only from configured channel regardless of `restrictAccess` flag

Comment (mahendrabagul) [reply pkg/execute/executor.go]: I tested the flow and its working now.

File: pkg/execute/executor.go (hunk 1 above)
Comment (mahendrabagul) [reply pkg/execute/executor.go]: This statement is not required now, so removed the message.

**Comment by PrasadG193** [comment]
Merged! Thanks @mahendrabagul for the PR

**Pull request merged**